test/integration/bugs_test.go

```diff
@@ -219,6 +226,32 @@ func TestArchiveContains(t *testing.T) {
 	checker.SinceNow().Search(`Recording events/openshift-monitoring`)
 	checker.EnableSinceLastCheck().Search(`Wrote \d+ records to disk in \d+`)
 
+	//https://bugzilla.redhat.com/show_bug.cgi?id=1868165
+	t.Run("Nodes", latestArchiveContainsNodes)
+
+	//https://bugzilla.redhat.com/show_bug.cgi?id=1881816
+	t.Run("MachineSet",
+		genLatestArchiveCheckPattern(
+			"machine set", matchingFileExists,
+			`^machinesets/.*\.json$`))
+
+	//https://bugzilla.redhat.com/show_bug.cgi?id=1881905
+	t.Run("PodDisruptionBudgets",
+		genLatestArchiveCheckPattern(
+			"pod disruption budgets", matchingFileExists,
+			`^config/pdbs/.*\.json$`))
+
+	t.Run("csr",
+		genLatestArchiveCheckPattern(
+			"csr", matchingFileExists,
+			`^config/certificatesigningrequests/.*\.json$`))
+
+	//https://bugzilla.redhat.com/show_bug.cgi?id=1879068
+	t.Run("HostsSubnet",
+		genLatestArchiveCheckPattern(
+			"hosts subnet", matchingFileExists,
+			`^config/hostsubnet/ip-.*\.json$`))
```

Comment:
Wouldn't be easier to use this without the `ip` prefix there? I think there will be always some hostsubnet resource on a cluster.